### ISSUE TITLE
Add Momentum Paper

### ISSUE THREAD

**Issue by bhavnicksm** [Dawn-Of-Eve/awesome-optimizers#14]
Ning Qian. On the momentum term in gradient descent learning algorithms. Neural networks :
the official journal of the International Neural Network Society, 12(1):145–151, 1999.
